assets/training/finetune_acft_multimodal/environments/acpt_multimodal/context/Dockerfile

```diff
@@ -6,4 +6,7 @@ RUN apt-get -y update
 
 # Install required packages from pypi
 COPY requirements.txt .
-RUN pip install -r requirements.txt --no-cache-dir
\ No newline at end of file
+RUN pip install -r requirements.txt --no-cache-dir
+
+# Vulnerability fix
+RUN pip install datasets==2.14.7
```

Comment:
Why do we need to install it here, if the correct version is added in requirements.txt

Reply:
It'll get overwritten by azureml-metrics package. azureml-metrics uses lower version of datasets which has vulnerability

Reply:
Thanks for the explanation. Should we add a comment in the Dockerfile about this ?